### DISCUSSION TITLE
Request about commercial purposes

### DISCUSSION THREAD

**Discussion by Linderjulia** [facebookresearch/seamless_communication#348] [General]
Hi, my name is Julia. I am working with an idea I have and found out about SeamlessM4T through a researcher from a university in Sweden. I wonder if there is any possibility to access SeamlessM4T for commercial purposes. Namely, I have an idea that I am trying to develop in Sweden for the municipalities here, to develop and facilitate the work in public organizations. I would like to have a meeting with you to tell you more if you are open to it.
Best regards, Julia Linder

**Comment by jmp84** [comment]
You can check the license details here: https://github.com/facebookresearch/seamless_communication?tab=readme-ov-file#license
Note that some artifacts allow commercial use while others do not allow it

**Comment by jmp84** [comment]
You can check the license details here: https://github.com/facebookresearch/seamless_communication?tab=readme-ov-file#license
Note that some artifacts allow commercial use while others do not allow it

**Reply by ShaneTeks** [reply]
I am also highly interested in getting commercial licenses for using the models. What is the procedure to acquire commercial licenses for these models?
Regards

**Reply by shashwat-aditya** [reply]
Did you ever get to figuring it out @ShaneTeks / @Linderjulia
